### PR TITLE
wait action executor

### PR DESCRIPTION
**Pull request by chengjoey**
## Description
add wait action executor

## Checklist

 - [ ] I made sure to check the compatibility of the erda version statemented in the action's spec.yml.
 - [ ] My change is adequately tested.

### CODE REVIEW

File: actions/wait/1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## 详细介绍
+wait action 主要是支持用户在执行流水线时需要等待一段时间再执行下一个action的功能
+
+## params
+### wait_time
+
+必填。
+等待的时间，单位为秒
+
```
Comment (sfwn) [inline actions/wait/1.0/README.md:13]: format your text. Some ends with two blank line, some one.

File: actions/wait/1.0/dice.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### job 配置项
+jobs:
+  wait:
+    image: registry.erda.cloud/erda-actions/wait:latest
```
Comment (sfwn) [inline actions/wait/1.0/dice.yml:4]: latest is not allowed.

File: actions/wait/1.0/spec.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+category: test_management
+displayName: 等待
+desc: 执行等待操作
+logoUrl:
+public: true
+executor:
+  kind: "WAIT"
+  name: "wait"
+labels:
+supportedVersions: # Deprecated. Please use supportedErdaVersions instead.
```
Comment (sfwn) [inline actions/wait/1.0/spec.yml:13]: remove these lines.

File: actions/wait/1.0/spec.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: wait
+version: '1.0'
+type: action
+category: test_management
```
Comment (sfwn) [inline actions/wait/1.0/spec.yml:4]: not just for test management.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: actions/wait/1.0/spec.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+logoUrl:
+public: true
+executor:
+  kind: "WAIT"
+  name: "wait"
+labels:
+supportedVersions: # Deprecated. Please use supportedErdaVersions instead.
+  - ">= 4.0"
+supportedErdaVersions:
+  - ">= 1.3"
```
Comment (sfwn) [inline actions/wait/1.0/spec.yml:16]: maybe 1.2?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: actions/wait/1.0/spec.yml (hunk 3 above)
Comment (chengjoey) [reply actions/wait/1.0/spec.yml:13]: image
kind and name to specify the executor(wait)

**Review by sfwn — APPROVED** [review]

**Pull request merged**